Manito/Manito/Screens/CreateRoom/Component/InputDateView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  InputDateView.swift
+//  Manito
+//
+//  Created by LeeSungHo on 2022/06/11.
+//
+
+import UIKit
+
+import SnapKit
+
+class InputDateView: UIView {
+    
+    // MARK: - Property
+    // 방 기한 설정 뷰
+    private let dateViewLabel: UILabel = {
+        let label = UILabel()
+        label.text = "활동 기간을 설정해 주세요"
+        label.font = .font(.regular, ofSize: 18)
+        return label
+    }()
+    
+    private let dateBackView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGray
+        view.layer.borderWidth = 1
+        view.layer.borderColor = UIColor.subOrange.cgColor
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    
+    private var dateLabel: UILabel = {
+        let label = UILabel()
+        label.text = "2022.06.06 ~ 2022.06.11"
+        label.font = .font(.regular, ofSize: 20)
+        return label
+    }()
+    
+    // MARK: - Init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Selector
+    @objc func didTapDateBackView(_ gesture: UITapGestureRecognizer){
+        print("gesture")
+    }
+    
+    // MARK: - Config
+    func render(){
+        
+        self.addSubview(dateViewLabel)
+        dateViewLabel.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.leading.trailing.equalToSuperview()
```

Comment:
p2;

```suggestion
            $0.top.leading.trailing.equalToSuperview()
```
이렇게 써도 되지 않나요

Reply:
예예예 맞습니당

Reply:
[22afe14](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/15/commits/22afe14508999ef63df4e3170311ba592f6bf2d7)